site/src/main/mdoc/driver_session_transaction.md

```diff
@@ -13,9 +13,18 @@ A `Driver` is basically the connection with the Database. Usually, you would onl
 A `Session` provides a context for performing operations _(`Transactions`)_ over the database. You may need as many as concurrent operations you want to have.
 A `Transaction` is a logical container for an atomic unit of work. Only one transaction may exist in a `Session` at any point in time.
 
-## Transaction managment
+## Session thread safety
 
-Each `Transaction` has to be started, used and finally either, committed or rolled back.
+Unlike its **Java** counterpart, `neotypes.Session` is thread safe.
+We achieve that by using a simple locking mechanism that ensures only one `Transaction` can be created `Session`.
```

Comment:
```
only one `Transaction` can be created `Session`
```
Should it be `one one Transaction at a time for each Session` or something like that